features/step_definitions/assertion_steps.rb

```diff
@@ -3,11 +3,15 @@
 end
 
 Then("I should see {string} in {string}") do |session, status|
-	within("##{status}") do
+	within("#{status}") do 
```

Comment:
Why did you change this? Are the trainee can see all sessions tests are passing?

Reply:
We have changed back to ##{status} to make the test passes